spec/FSi/Bundle/ResourceRepositoryBundle/Repository/Resource/Type/CKEditorTypeSpec.php

```diff
@@ -86,4 +86,14 @@ function it_return_form_builder_with_form_options_added_to_resource_definition(F
 
         $this->getFormBuilder($factory)->shouldReturnAnInstanceOf(FormBuilder::class);
     }
+
+    /**
+     * @return string
+     */
+    private function getCKEditorFormType()
```

Comment:
I believe this method is never used because version `2.0` of `fsi/resource-repository-bundle` will require `symfony/form` in version `^3.0|^4.0`.